tools/yaml-dumper/yaml-dumper.go

```diff
@@ -0,0 +1,50 @@
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"kubevirt.io/hostpath-provisioner-operator/tools/helper"
+	"kubevirt.io/hostpath-provisioner-operator/tools/util"
+)
+
+var (
+	exportPath = flag.String("export-path", "", "")
+)
+
+// Export the CDI CRDs schemas from code to yaml.
```

Comment:
you mean HPP CRDs?

Reply:
Yeah, I wasn't going to include this, but if at some point I decide I want to ensure the code and deploy/operator.yaml are in sync, having this will be easy. I added a PR to fix the comment.